service-front/module/Application/view/application/general/cookies/index.twig

```diff
@@ -1,44 +1,50 @@
 {% extends 'layout/layout.twig' %}
-
 {%- set pageTitle = 'Cookies' -%}
-
 {% import "application/macros.twig" as macros %}
-
 {% set usageCookies = form.get('usageCookies') %}
-
 {% block content %}
 
     <p>Cookies are files saved on your phone, tablet or computer when you visit a website.</p>
-
     <p>
         We use cookies to store information about how you use the Make a lasting power of attorney service, such as the
         pages you visit.
     </p>
 
-
     {# begin form #}
     {{ form.prepare() ? '' }}
     {{ form().openTag( form )|raw }}
         {{ formElement(form.getCsrf) }}
 
-        <h2 class="heading-large">Cookie settings</h2>
+    <h2 class="heading-large">Change your cookie settings</h2>
 
-        <p>We use two types of cookie. One type is strictly necessary and the other type of cookie is optional.</p>
+    <p>We use two types of cookie. One type is strictly necessary and the other type of cookie is optional.</p>
 
-        <h3 class="heading-medium">Cookies that measure website use</h3>
+    <h3 class="heading-medium">Strictly necessary cookies</h3>
```

Comment:
Suggest we raise an issue to modify the content so we enumerate the cookies in tables, as per the cookies page guidance.